test/protobuf/protoc/proto_gen/extension3.pb.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Ext.MyEventMessage do
+  @moduledoc false
+  use Protobuf, custom_field_options?: true, syntax: :proto3
+
+  @type t :: %__MODULE__{
+          f1: float | nil
+        }
+
+  defstruct [:f1]
+
+  def full_name do
+    "ext.MyEventMessage"
+  end
+
+  def message_options do
+    # credo:disable-for-next-line
+    "[\"{\\\"is_event\\\":true}\"]"
```

Comment:
Is there a way to turn this back into elixir rather than it be an escaped string? Something like 
```suggestion
    [{"is_event", true}]
```

Reply:
I tried but it only allows string in this function here :/

Reply:
now it returns:
```
  def message_options do
    # credo:disable-for-next-line
    [%{is_event: true}]
  end
```